examples/core-auth-nextjs-pages/src/pages/_app.tsx

```diff
@@ -40,7 +39,7 @@ const NAVIGATION: Navigation = [
 ];
 
 const BRANDING = {
-  title: 'My Toolpad Core App',
+  title: 'My Toolpad Core Next.js Pages App',
```

Comment:
I wouldn't do this.

Reply:
It makes the app easier to identify while testing/running - in all likelihood a person running this example is going to change these anyway regardless of whatever the default text is

Reply:
> It makes the app easier to identify while testing/running

Sure, but that's not a problem our users would have. We can't do it for all permutations, so I propose we do it for none.

Reply:
I propose we only do it for things that aren't immediately visible on the app (Auth providers/Theme/Branding are all visible and obvious). Currently it would only be the type of router; if we introduce something that fits this criteria in the future, we could do it for that as well

Reply:
The previous title was already looking a bit too long for the current layout, I would also prefer to use shorter titles...

Reply:
Removed the longer titles